index.html

```diff
@@ -4065,7 +4065,7 @@ <h3>Parsing content</h3>
     </p>
     <p>
       Web NFC does not sign <a>NFC content</a>. Using <a>NDEF signature</a> and
-      key management is application domain.
+      key management is application's responsibility.
```

Comment:
is the responsibility of the application

Reply:
amended